### PR TITLE
Basic program to find perimeter of the recatangle!

### PR DESCRIPTION
**Pull request by MayankJain10**
Perimeter of rectangle ABCD = 2A + 2B = 2(A+B)

### CODE REVIEW

File: src/main/java/org/mayank/basicPrograms/Basics.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+     * Perimeter of rectangle ABCD = 2A + 2B = 2(A+B)
+     * The perimeter of a rectangle is the total length of all its four sides.
+     * It can be simply calculated by all its four sides.
+     */
+
+    static void perimeter(int length, int breadth) {
+        // calculate the perimeter using formula
+         int perimeter = 2 * (length + breadth);
+
+        System.out.println("The perimeter of the given rectangle of length" + length +
```
Comment (harshPareek-git) [inline src/main/java/org/mayank/basicPrograms/Basics.java:23]: System.out.println("The perimeter of the given rectangle of length" + " " + length +
        " and breadth" + " "+ breadth + " = " + perimeter);

**Review by harshPareek-git — CHANGES_REQUESTED** [review]
Please run it before submission

**Comment by MayankJain10** [comment]
> Please run it before submission

already run the program and tested!!

File: src/main/java/org/mayank/basicPrograms/Basics.java (hunk 1 above)
Comment (MayankJain10) [reply src/main/java/org/mayank/basicPrograms/Basics.java:23]: What changes required!?

**Comment by MayankJain10** [comment]
**Test results:**
Enter the length of the triangle: 
11
Enter the breadth of the triangle: 
22
The perimeter of the given rectangle of length11and breadth22 = 66

Process finished with exit code 0

File: src/main/java/org/mayank/basicPrograms/Basics.java (hunk 1 above)
Comment (harshPareek-git) [reply src/main/java/org/mayank/basicPrograms/Basics.java]: Some spacing issues were left which are resolved now!

**Review by harshPareek-git — APPROVED** [review]

**Pull request merged**